packages/marko-web-theme-monorail/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@parameter1/base-cms-marko-web-theme-monorail",
+  "version": "2.88.1",
+  "author": "Brian Miller <brian@parameter1.com>",
+  "repository": "https://github.com/parameter1/base-cms/tree/master/packages/marko-web-theme-monorail",
+  "license": "MIT",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "lint": "eslint --ext .js --ext .vue --max-warnings 5 --ignore-path ../../.eslintignore ./",
+    "test": "yarn lint"
+  },
+  "dependencies": {
+    "@parameter1/base-cms-image": "^2.81.0",
+    "@parameter1/base-cms-inflector": "^2.0.0",
+    "@parameter1/base-cms-marko-core": "^2.80.0",
+    "@parameter1/base-cms-marko-web": "^2.81.0",
+    "@parameter1/base-cms-marko-web-deferred-script-loader": "^2.77.0",
+    "@parameter1/base-cms-marko-web-gam": "^2.77.0",
+    "@parameter1/base-cms-marko-web-gtm": "^2.88.0",
+    "@parameter1/base-cms-marko-web-html-sitemap": "^2.80.0",
+    "@parameter1/base-cms-marko-web-icons": "^2.46.3",
+    "@parameter1/base-cms-marko-web-identity-x": "^2.88.1",
+    "@parameter1/base-cms-marko-web-inquiry": "^2.75.1",
+    "@parameter1/base-cms-marko-web-native-x": "^2.88.0",
+    "@parameter1/base-cms-marko-web-omeda": "^2.79.0",
+    "@parameter1/base-cms-marko-web-omeda-identity-x": "^2.88.1",
+    "@parameter1/base-cms-marko-web-p1-events": "^2.77.0",
+    "@parameter1/base-cms-marko-web-search": "^2.80.0",
+    "@parameter1/base-cms-marko-web-social-sharing": "^2.75.1",
+    "@parameter1/base-cms-marko-web-theme-default": "^2.75.1",
+    "@parameter1/base-cms-object-path": "^2.75.1",
+    "@parameter1/base-cms-utils": "^2.75.1",
+    "@parameter1/base-cms-web-cli": "^2.81.0",
+    "@parameter1/base-cms-web-common": "^2.80.0",
+    "@parameter1/omeda-graphql-client-express": "^0.4.2",
+    "body-parser": "^1.19.0",
+    "cheerio": "^1.0.0-rc.10",
+    "express": "^4.17.1",
+    "graphql": "^14.5.4",
+    "graphql-tag": "^2.10.1",
+    "html-entities": "^2.1.0",
+    "http-errors": "^1.7.3",
+    "moment": "^2.29.1",
+    "newrelic": "^6.4.1",
+    "node-fetch": "^2.6.1",
+    "object-path": "^0.11.5",
+    "slug": "^4.0.2"
```

Consolidating comments: The default theme package should be removed, along with direct dependencies for `web-cli` `omeda-graphql-client-express`, and other third-party packages (`body-parser`, `express`, `graphql`, `html-entities`, `newrelic`, `object-path`, and `slug`).

```suggestion
    "@parameter1/base-cms-object-path": "^2.75.1",
    "@parameter1/base-cms-utils": "^2.75.1",
    "@parameter1/base-cms-web-common": "^2.80.0",
    "cheerio": "^1.0.0-rc.10",
    "graphql-tag": "^2.10.1",
    "http-errors": "^1.7.3",
    "moment": "^2.29.1",
    "node-fetch": "^2.6.1"
```